README.md

```diff
@@ -171,6 +171,8 @@ git clone https://github.com/chipsalliance/chisel3.git
 cd chisel3
 sbt compile
 ```
+In order to run the following unit tests, make sure you have Verilator installed and loaded
+(Check this by running 'which verilator')
```

Comment:
I think this README change was already performed in a separate PR. Can you remove it from this PR to keep it clean?

Reply:
you may need to rebase your changes or merge in latest master